05/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta
+        name="viewport"
+        content="width=device-width, initial-scale=1.0"
+    >
+
+    <link
+        rel="stylesheet"
+        href="./css/reset.css"
+    >
+    <link
+        rel="stylesheet"
+        href="./css/main.css"
+    >
+    <link
+        rel="stylesheet"
+        media="(min-width: 800px)"
+        href="./css/desktop.css"
+    >
+    <script
+        src="https://kit.fontawesome.com/1611b70a0e.js"
+        crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
+</head>
+
+<body>
+
+    <header class="header container">
+        <div class="header__container content">
+            <h2 class="header__logo content__item"></h2>
+            <nav class="header__navbar content__item">
+                <i class="burger fa-solid fa-bars"></i>
```

Comment:
Warto ukryć "hamburgera" w wersji mobilnej :)

Reply:
Chodzi pewnie o wersje desktopową, w sumie to nie wiem co się stało, bo burger miał 'display: none'